pkg/clusteragent/admission/mutate/agent_sidecar/agent_sidecar.go

```diff
@@ -86,7 +92,7 @@ func getDefaultSidecarTemplate() *corev1.Container {
 			},
 			{
 				Name:  "DD_CLUSTER_NAME",
-				Value: config.Datadog.GetString("cluster_name"),
+				Value: clustername.GetClusterName(context.TODO(), hname),
```

Comment:
can we get it from the config package?

Reply:
The `config` package can only get the cluster name explicitly manually set in the config. It cannot get what is automatically detected.